docs/reference/connectors/http.md

```diff
@@ -63,147 +63,32 @@ end;
 ```tremor title="config.troy"
 define connector `http-in` from http_server
 with
-  codec = "json",
   config = { 
     "url": "http://localhost:8080",
 
     # Optional Transport Level Security configuration
-    # "tls" = { ... },
+    # "tls": { ... },
 
     # Custom Mime Codec Map, overrides default `codec`
-    # "custom_codecs" = {
+    # "mime_mapping": {
     #    # key defines the MIME type, value defines codec by name 
     #    "application/json": "json",
     #    "application/yaml": "yaml"
+    #    "*/*": "json"               # default codec
     # }
   }
 end;
 ```
 
-## HTTP configuration example
+## Codecs
 
-This is a relatively basic client server system that replays JSON formatted lines of data from a text file over HTTP to a server. The
-server receives the JSON events and echo's them back to the HTTP client.
-
-The client and server are implemented as tremor flows.
-
-A high level summary of the overall flow:
-
-```mermaid
-graph LR
-    A[JSON File] -->|read line by line| B(HTTP Client)
-    B -->|send json request| C{HTTP Server}
-    C{HTTP Server} -->|receive json request| D(select event from in into out)
-    D -->|echo json response| B{HTTP Client}
-    B -->|log response| E[Log File]
-```
+The HTTP connectors are somewhast special as they handle the encoding and decoding based on the content-type header not by a single defined codec.
 
-### The complete annotated source
+By default tremor supplies a  mapping of mime-type to it's codecs as part of the base implementation. However it is possible to overwrite this with the `mime_mapping` option.
```

Comment:
It would be nice to give some usage examples on how to specify the `mime_mapping` and bring codecs to effect. Maybe with an example incoming request?

Reply:
good call, added a section!